assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.stories.js

```diff
@@ -66,6 +66,12 @@ const topCitiesReportOptions = {
 	endDate: '2024-03-27',
 	startDate: '2024-02-29',
 	dimensions: [ 'city' ],
+	dimensionFilters: {
+		city: {
+			filterType: 'emptyFilter',
+			notExpression: true,
+		},
+	},
```

Comment:
This will need reverting as per my comment on `assets/js/modules/analytics-4/hooks/useAudienceTilesReports.js`.

```suggestion
```